lms/djangoapps/grades/models.py

```diff
@@ -742,14 +738,6 @@ def update_or_create_override(
             override_history_entry.history_user = requesting_user
             override_history_entry.save()
 
-        action = action or PersistentSubsectionGradeOverrideHistory.CREATE_OR_UPDATE
-
-        PersistentSubsectionGradeOverrideHistory.objects.create(
-            override_id=override.id,
-            user=requesting_user,
-            feature=feature,
```

Comment:
just to double-check: the state of the `system` field does get recorded via the simple history table, right?

Reply:
Yes.